AuthenticatorShared/Core/Vault/Repositories/AuthenticatorItemRepositoryTests.swift

```diff
@@ -202,8 +263,65 @@ class AuthenticatorItemRepositoryTests: AuthenticatorTestCase {
         )
     }
 
-    /// `itemListPublisher()` returns a favorites section.
+    /// `itemListPublisher()` returns a favorites section (when the feature flag is Off)
```

Comment:
⛏️ We describe it as `enabled` in the next test; in my opinion, the comments on each should mirror each other to make the parallel behavior more obvious

Reply:
👍  Good note, I've aligned them in the latest commit.